report_api/app/report.py

```diff
@@ -54,16 +62,16 @@ def constructQuery(j, query):
 
     return my_query
 
-def queryElastic(login_query):
+def queryElastic(login_query, client_ip):
     try:
         response = elastic.search(index=app.config['ELASTICSEARCH_INDEX'],
                                   body=login_query,
                                   _source=True)
     except elasticsearch.TransportError as err:
-        print("Elasticsearch transport error: ", err.error)
+        app.logger.error("Elasticsearch transport error: %s remote_ip=%s", err.error, client_ip)
         return None
     except elasticsearch.ElasticsearchException:
-        print("Elasticsearch exception")
+        app.logger.error("Elasticsearch Exception error remote_ip=%s", client_ip)
```

Comment:
It turns out that the second exception is indeed generic exception, and that all the exceptions which include an error will be in the previous exception class TransportError.